account_move_base_import/views/account_move_view.xml

```diff
@@ -64,4 +64,22 @@
 
     <menuitem string="Move Completion Rule" action="action_move_completion_rule_tree"
               id="menu_action_move_completion_rule_tree_menu" parent="account.account_management_menu"/>
+
+    <record id="view_account_move_filter" model="ir.ui.view">
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_account_move_filter"/>
+        <field name="arch" type="xml">
+            <separator position="after">
+                <filter
```

Comment:
you should add that for migration ?

Reply:
Good catch ! @damdam-s can you drop the related commit ?